apps/web/src/services/administration/components/UserManagementTable.vue

```diff
@@ -326,7 +326,7 @@ export default {
             });
         };
         const unbindRole = async (userId) => {
-            const { results } = await SpaceConnector.clientV2.identity.roleBinding.list<RoleListParameters, ListResponse<RoleBindingModel>>({
+            const { results } = await SpaceConnector.clientV2.identity.roleBinding.list<RoleBindingListParameters, ListResponse<RoleBindingModel>>({
```

Comment:
nitpick: RoleBindingListResponse ??
It seems like a discussion is needed. How should ListResponse be used?
1. ListResponse\<RoleBindingModel\>
2. [RoleBindingListResponse](https://github.com/cloudforet-io/console/pull/2371/commits/0cc50a7d587e5dc6950284bf9cf18d889fe859a0#diff-39067320d718e8cefb8085767b2b63b7576dc95d8a492bfba1903cceb2fe5e36)